cfe_internal/CFE_hub_specific.cf

```diff
@@ -249,7 +249,7 @@ bundle edit_line change_port(p)
 
     any::
 
-      "^Listen(\s+)(?!$(p)).*"
+      "^\s?Listen(\s+)(?!$(p)$).*"
```

Comment:
Also, doesn't the the final ``$`` contradict the ``.*`` that follows it? I suggest:

<pre>
^\s*Listen(\s+)(?!$(p))\s*$
</pre>

Which makes it even more robust by matching trailing whitespace as well. I think the regex is anchored, but good to have ``$`` at the end still.